recipe/meta.yaml

```diff
@@ -17,6 +17,8 @@ requirements:
     - jupyterhub-singleuser =1.4.2
     - jupyterlab =3.1.10
     - nbgitpuller =1.0.2
+    - rise
```

Comment:
see comment below, i would suggest an explicit pin here for consistency with the other packages